functional-tests/test-bed/src/test/java/org/arquillian/smart/testing/ftest/configuration/SkipTestExecutionFunctionalTest.java

```diff
@@ -101,4 +107,103 @@ public void should_not_install_extension_when_only_clean_goal_is_used() throws E
             .doesNotContainDirectory(SMART_TESTING_WORKING_DIRECTORY_NAME)
             .doesNotContainDirectory("target");
     }
+
+    @Test
+    public void should_disable_smart_testing_and_execute_no_tests_when_skip_is_set_in_plugin_configuration_section_of_pom() throws Exception {
+        // given
+        final Project project = testBed.getProject();
+
+        project
+            .applyAsCommits("Configure skipTests property in plugin configuration section of pom");
+
+        project.configureSmartTesting()
+                .executionOrder(AFFECTED)
+                .inMode(ORDERING)
+            .enable();
+
+        // when
+        final TestResults actualTestResults = project
+            .build(CORE_MODULES)
+                .options()
+                    .withDebugOutput()
+                    .logBuildOutput(false)
+                .configure()
+            .run();
+
+        // then
+        String capturedMavenLog = project.getMavenLog();
+        softly.assertThat(capturedMavenLog).contains("[DEBUG] Smart Testing Extension - Disabling Smart Testing ");
+        softly.assertThat(actualTestResults.accumulatedPerTestClass()).size().isEqualTo(0);
+    }
+
+    @Test
+    public void should_execute_only_unit_tests_when_skipITs_is_set_in_pom() throws Exception {
+        // given
+        final Project project = testBed.getProject();
+
+        project
+            .applyAsCommits("Configure skipITs as default property in pom");
+
+        project.configureSmartTesting()
+                .executionOrder(AFFECTED)
+                .inMode(ORDERING)
+            .enable();
+
+        // when
+        final TestResults actualTestResults = project.build(CORE_MODULES).run("clean", "verify");
+
+        // then
+        String capturedMavenLog = project.getMavenLog();
+        softly.assertThat(capturedMavenLog).contains(SMART_TESTING_EXTENSION_ENABLED);
+        softly.assertThat(actualTestResults.accumulatedPerTestClass()).size().isEqualTo(20);
+    }
+
+    @Test
+    public void should_execute_only_unit_tests_when_skipITs_is_set_as_custom_property_in_pom() throws Exception {
+        // given
+        final Project project = testBed.getProject();
+
+        project
+            .applyAsCommits("Configure skipITs as custom default property in pom");
+
+        project.configureSmartTesting()
+                .executionOrder(AFFECTED)
+                .inMode(ORDERING)
+            .enable();
+
+        // when
+        final TestResults actualTestResults = project.build(CORE_MODULES).run("clean", "verify");
+
+        // then
+        String capturedMavenLog = project.getMavenLog();
+        softly.assertThat(capturedMavenLog).contains(SMART_TESTING_EXTENSION_ENABLED);
+        softly.assertThat(actualTestResults.accumulatedPerTestClass()).size().isEqualTo(20);
+    }
+
+    @Test
+    public void should_be_able_to_enable_tests_from_property_when_set_as_default_property_in_pom() throws Exception {
```

Comment:
I don't think that this test is really needed - you just verify the behavior of Maven - creating the effective pom files.